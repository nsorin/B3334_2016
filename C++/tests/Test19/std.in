S seg1 0 20 10 0
R rect1 0 200 100 10
R rect2 0 10 10 0
OR reu1 rect1 rect2 seg1
HIT seg1 5 10
HIT seg1 5 5
MOVE seg1 5 5
HIT seg1 10 15
HIT seg1 10 10
HIT reu1 5 20
HIT rect1 5 20
HIT rect2 5 20
MOVE reu1 1000 1000
HIT reu1 5 20
HIT reu1 10 20
DELETE seg1
HIT reu1 10 20
EXIT

