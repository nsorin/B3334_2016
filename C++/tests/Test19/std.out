#S seg1 0 0 10 20
OK
#R rect1 0 10 100 200
OK
#R rect2 0 0 10 10
OK
#OR reu1 rect1 rect2 seg1
OK
#HIT seg1 5 10
YES
#HIT seg1 5 5
NO
#MOVE seg1 5 5
OK
#HIT seg1 10 15
YES
#HIT seg1 10 10
NO
#HIT reu1 5 20
YES
#HIT rect1 5 20
YES
#HIT rect2 5 20
NO
#MOVE reu1 1000 1000
OK
#HIT reu1 5 20
NO
#HIT reu1 10 20
NO
#DELETE seg1
OK
#HIT reu1 10 20
NO
